### ISSUE TITLE
The `#shelve` button does not have a name available to an accessibility API

### ISSUE THREAD

**Issue by jeremyfelt** [washingtonstateuniversity/WSU-spine#404]
We should probably have a title attribute assigned explaining that this is the menu button.

See: https://www.w3.org/TR/WCAG20-TECHS/H91
